driver/tke_driver.go

```diff
@@ -618,10 +620,19 @@ func (d *Driver) PostCheck(ctx context.Context, info *types.ClusterInfo) (*types
 
 // Remove implements driver remove interface
 func (d *Driver) Remove(ctx context.Context, info *types.ClusterInfo) error {
+	logrus.Info("invoking removeCluster")
 	state, err := getState(info)
 	if err != nil {
 		return err
 	}
+
+	// Validate if the cluster info has clusterID
+	err = ReflectStructField(state, "ClusterID")
```

Comment:
I don't think this function is necessary. We can check it by
```
if state ==nil || state.ClusterID ==""
```

Reply:
updated.